app/views/spree/admin/images/edit.html.haml

```diff
@@ -7,7 +7,7 @@
 
 = form_for [:admin, @product, @image], url: admin_product_image_path(@product, @image, @url_filters), html: { multipart: true } do |f|
   %fieldset
-    %legend{align: "center"}= @image.attachment_file_name
+    %legend{align: "center"}= @image.attachment&.filename
```

Comment:
Active Storage always provides an `attachment` object. You can simplify this:
```suggestion
    %legend{align: "center"}= @image.attachment.filename
```

But it doesn't really matter.

```
[1] pry(main)> Spree::Image.new.attachment
=> #<ActiveStorage::Attached::One:0x00005600e4f32998
 @name="attachment",
 @record=
  #<Spree::Image:0x00005600e495ab88
   id: nil,
   viewable_id: nil,
   attachment_width: nil,
   attachment_height: nil,
   attachment_file_size: nil,
   position: nil,
   viewable_type: nil,
   attachment_content_type: nil,
   attachment_file_name: nil,
   type: "Spree::Image",
   attachment_updated_at: nil,
   alt: nil>>
[2] pry(main)> Spree::Image.new.attachment.filename
=> nil
```

Reply:
Oh, Sure @mkllnk

Reply:
Done!